charts/jenkins/pod-templates/nuxeo-web-ui-ftests.yaml.gotmpl

```diff
@@ -19,6 +19,18 @@
     resourceRequestMemory: "4Gi"
     ttyEnabled: {{ .Values.podTemplates.ttyEnabled }}
     workingDir: {{ .Values.podTemplates.workingDir }}
+  - args: ""
+    command: ""
+    image: mcr.microsoft.com/playwright:v1.13.1
```

Comment:
Finally, according to https://jira.nuxeo.com/browse/NXBT-3634?focusedCommentId=617883&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-617883, we want to keep 1.13.1, otherwise it doesn't work.

Reply:
Thanks 👍